Resources/Maps/Shuttles/bulkermax.yml

```diff
@@ -0,0 +1,4880 @@
+    - color: '#FF1212FF'
+      type: AtmosPipeColor
+  - uid: 222
+    components:
+    - rot: 3.141592653589793 rad
+      pos: 0.5,-3.5
+      parent: 1
+      type: Transform
+    - color: '#FF1212FF'
+      type: AtmosPipeColor
+- proto: GravityGeneratorMini
+  entities:
+  - uid: 94
+    components:
+    - pos: -10.5,1.5
+      parent: 1
+      type: Transform
+- proto: Grille
+  entities:
+  - uid: 341
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: 2.5,4.5
+      parent: 1
+      type: Transform
+  - uid: 428
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -1.5,2.5
+      parent: 1
+      type: Transform
+  - uid: 466
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: 0.5,4.5
+      parent: 1
+      type: Transform
+  - uid: 467
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -0.5,4.5
+      parent: 1
+      type: Transform
+  - uid: 468
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: 1.5,4.5
+      parent: 1
+      type: Transform
+  - uid: 496
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: 2.5,2.5
+      parent: 1
+      type: Transform
+  - uid: 563
+    components:
+    - rot: 1.5707963267948966 rad
+      pos: -1.5,4.5
+      parent: 1
+      type: Transform
+- proto: Gyroscope
+  entities:
+  - uid: 98
+    components:
+    - pos: -9.5,1.5
+      parent: 1
+      type: Transform
+- proto: HospitalCurtainsOpen
+  entities:
+  - uid: 625
+    components:
+    - rot: -1.5707963267948966 rad
+      pos: 9.5,-2.5
+      parent: 1
+      type: Transform
+  - uid: 626
+    components:
+    - rot: -1.5707963267948966 rad
+      pos: 10.5,-2.5
+      parent: 1
+      type: Transform
+  - uid: 627
+    components:
+    - rot: -1.5707963267948966 rad
+      pos: 10.5,-4.5
+      parent: 1
+      type: Transform
+  - uid: 628
+    components:
+    - rot: -1.5707963267948966 rad
+      pos: 9.5,-4.5
+      parent: 1
+      type: Transform
+- proto: HydroponicsToolClippers
+  entities:
+  - uid: 273
+    components:
+    - pos: -7.640341,-2.51385
+      parent: 1
+      type: Transform
+- proto: HydroponicsToolMiniHoe
+  entities:
+  - uid: 272
+    components:
+    - pos: -7.5690746,-2.4322877
+      parent: 1
+      type: Transform
+- proto: hydroponicsTrayAnchored
+  entities:
+  - uid: 66
+    components:
+    - rot: 3.141592653589793 rad
+      pos: -10.5,-3.5
+      parent: 1
+      type: Transform
+  - uid: 67
+    components:
+    - rot: 3.141592653589793 rad
+      pos: -10.5,-2.5
+      parent: 1
+      type: Transform
+  - uid: 140
+    components:
+    - rot: 3.141592653589793 rad
+      pos: -10.5,-4.5
+      parent: 1
+      type: Transform
+- proto: KitchenMicrowave
+  entities:
+  - uid: 285
+    components:
+    - pos: -8.5,-2.5
+      parent: 1
+      type: Transform
+- proto: KitchenReagentGrinder
+  entities:
+  - uid: 156
+    components:
+    - pos: -9.5,-2.5
+      parent: 1
+      type: Transform
+- proto: LockerCaptain
```

Comment:
Please use the pre filled locker, this is harder for us to keep balance on ships by using ones you fill yourself

Reply:
Gotcha.